src/Bridge.ts

```diff
@@ -615,9 +615,9 @@ export class Bridge {
             (data) => connManager.getConnectionsForFeedUrl(data.feed.url),
             (c, data) => c.handleFeedEntry(data),
         );
-        this.bindHandlerToQueue<FeedEntry, FeedConnection>(
+        this.bindHandlerToQueue<{url: string}, FeedConnection>(
```

Comment:
For future-proofing, how about defining this as an interface, like `FeedSuccess`?  I have it written ready if interested; not a blocker if a release is time-critical.

Reply:
Yeah, I think we should. Uno momento.